rlclientlib/logger/async_batcher.h

```diff
@@ -136,11 +144,12 @@ namespace reinforcement_learning { namespace logger {
   int async_batcher<TEvent, TSerializer>::fill_buffer(
                                                       std::shared_ptr<utility::data_buffer>& buffer,
                                                       size_t& remaining,
-                                                      api_status* status)
+                                                      api_status* status,
```

Comment:
it looks like `original_count` can be passed in as a reference here instead of a pointer (no need to dereference or do a nullptr check)

Reply:
passing as reference instead of pointer